pkgs/applications/misc/megasync/default.nix

```diff
@@ -1,75 +1,91 @@
-{ lib
-, stdenv
-, autoconf
-, automake
-, c-ares
-, cryptopp
-, curl
-, doxygen
-, fetchFromGitHub
-, ffmpeg
-, libmediainfo
-, libraw
-, libsodium
-, libtool
-, libuv
-, libzen
-, lsb-release
-, mkDerivation
-, pkg-config
-, qtbase
-, qttools
-, qtx11extras
-, sqlite
-, swig
-, unzip
-, wget
+
+{
+  lib,
+  stdenv,
+  c-ares,
+  cmake,
+  cryptopp,
+  curl,
+  fetchFromGitHub,
+  ffmpeg,
+  freeimage,
+  hicolor-icon-theme,
+  icu,
+  libmediainfo,
+  libsodium,
+  libtool,
+  libuv,
+  libxcb,
+  libzen,
+  mkDerivation,
+  openssl,
+  pkg-config,
+  qtbase,
+  qtdeclarative,
+  qtgraphicaleffects,
+  qttools,
+  qtquickcontrols,
+  qtquickcontrols2,
+  qtsvg,
+  qtx11extras,
+  readline,
+  sqlite,
+  unzip,
+  wget,
+  zlib,
+  qt5,
 }:
 mkDerivation rec {
   pname = "megasync";
-  version = "4.9.1.0";
+  version = "5.5.0.0";
 
   src = fetchFromGitHub {
     owner = "meganz";
     repo = "MEGAsync";
     rev = "v${version}_Linux";
-    hash = "sha256-Y1nfY5iP64iSCYwzqxbjZAQNHyj4yVbSudSInm+yJzY=";
+    hash = "sha256-f+FZdMMdGZvYvJZ0xTUuO9zk0/VHFAISvM2cW0zwfaE=";
     fetchSubmodules = true;
   };
 
   nativeBuildInputs = [
-    autoconf
-    automake
-    doxygen
+    cmake
     libtool
-    lsb-release
     pkg-config
     qttools
-    swig
     unzip
   ];
   buildInputs = [
     c-ares
     cryptopp
     curl
     ffmpeg
+    freeimage
```

Comment:
Is freeimage really required? It's marked as insecure.

Reply:
No. I don't think so, it's "used to manage thumbnails/previews" (https://github.com/meganz/sdk/blob/master/examples/win32/README.md?plain=1#L50). I'll disable it.